starport/templates/typed/basic/stargate/component/proto/{{moduleName}}/{{typeName}}.proto.plush

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package <%= formatOwnerName(OwnerName) %>.<%= AppName %>.<%= ModuleName %>;
+
+option go_package = "<%= ModulePath %>/x/<%= ModuleName %>/types";
+
+import "gogoproto/gogo.proto";
+
+message <%= TypeName.UpperCamel %> {
+  string creator = 1;<%= for (i, field) in Fields { %>
```

Comment:
We don't need `creator` since since no interaction message is scaffolded

I just remembered that we should actually not generate it for other types when `no-message` is provided as well

Reply:
solved by 9204e56.